components/NotificationSnackBar.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <v-snackbar v-model="showSnackBar">
+        {{snackbarText}}
+        <template v-slot:action="{ attrs }">
+            <v-btn color="blue lighten-3" text v-bind="attrs" @click="() => {$emit('closeSnackbar')}">
```

Comment:
see earlier comment about updating the event name


edit: actually.....
1) shouldn't v-snackbar already have a built in close button? No need to add a second. We should just get rid of this button + event IMO. 
2) There's no need to emit the button click honestly....the consumer only cares about the event where the snackbar `closed` which is already handled with `v-model="showSnackBar"`

Reply:
I emitted the event for redirections cases,
In case theres a redirection to the home page, i choose to do it after the Ok! button has been clicked. 
Let me know your thought...

Reply:
your thinking makes sense, but the parent can use the `v-model` change for that. Then they can check if it was closed by a timeout OR a button click.